simpleclient_log4j/src/main/java/io/prometheus/client/log4j/InstrumentedAppender.java

```diff
@@ -0,0 +1,93 @@
+package io.prometheus.client.log4j;
+
+import io.prometheus.client.CollectorRegistry;
+import io.prometheus.client.Counter;
+import org.apache.log4j.Appender;
+import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.Level;
+import org.apache.log4j.spi.LoggingEvent;
+
+
+public class InstrumentedAppender extends AppenderSkeleton {
+    
+	private final CollectorRegistry registry;
+
+    private Counter all;
+    private Counter trace;
+    private Counter debug;
+    private Counter info;
+    private Counter warn;
+    private Counter error;
+    private Counter fatal;
+
+    /**
+     * Create a new instrumented appender using the default registry.
```

Comment:
This won't work if someone instantiates two of this class. I'd say for the initial version that you instrument in the usual way with static variables and without the registry being configurable. If other use cases come up, we can figure out how to do this properly then.